ecsmanage/management/commands/ecsmanage.py

```diff
@@ -170,34 +171,29 @@ def run_task(self, config, task_def_arn, security_group_id, subnet_id, cmd):
             taskDefinition=task_def_arn
         )["taskDefinition"]
 
+        kwargs = {
```

Comment:
Using a `kwargs` dictionary here aids in removing the `task_response` code duplication below.

Reply:
Nice refactor! 🎯